src/Hosting/Hosting/src/Host.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Reflection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.Hosting
+{
+    public static class Host
+    {
+        public static IHostBuilder CreateDefaultBuilder() =>
+            CreateDefaultBuilder(args: null);
+
+        public static IHostBuilder CreateDefaultBuilder(string[] args)
+        {
+            var builder = new HostBuilder();
+
+            builder.UseContentRoot(Directory.GetCurrentDirectory());
```

Comment:
Do we need the check we have in the other logic for the env var override?

Reply:
No, Host builder doesn't include env var config by default. We add it afterwards and it overrides.

Reply:
OK just make sure the scrnaeio fixed is indeed fixed here as well.